ndk/utils.tsx

```diff
@@ -406,7 +406,11 @@ export const createZapRequest = async ({
     filter: "writable",
   });
```

Comment:
could store these with redux

Reply:
wtf thats not the code i linked

Reply:
was supposed to be userRelayUrls

Reply:
i think using the in memory cache would be better than redux for this. i'll look into it, but not sure if we need that optimization right now.